fintorch/datasets/causal_data.py

```
@@ -0,0 +1,704 @@
+- get_dataset_info(): Get dataset information
+- get_clean_adjacency_matrix(): Extract clean adjacency matrix from groundtruth
+
+Performance Benefits of Polars:
+- 2-10x faster CSV reading compared to Pandas
+- Significantly lower memory usage for large datasets
+- Better handling of data types and null values
+- More intuitive and consistent API for data operations
+"""
+
+import os
+import zipfile
```

Ruff (0.11.9)

`zipfile` imported but unused

Remove unused import: `zipfile`

(F401)